packages/core/src/filter-set.ts

```diff
@@ -1,16 +1,10 @@
 import { Filters, FilterFn, FilterObject, FilterParams, InteractorSpecification } from './specification';
 import { noCase } from 'change-case';
 import { matcherDescription } from './matcher';
+import { MaybeMatcher } from '../dist';
```

Comment:
```suggestion
import { MaybeMatcher, matcherDescription } from './matcher';
```

Reply:
This was causing the build to fail. Looks like maybe VSCode was importing this automatically.